src/ralph/defaults.py

```diff
@@ -52,7 +53,7 @@ def load_config(config_file_path):
     """Return a dictionary representing Ralph's configuration."""
 
     try:
-        with open(config_file_path) as config_file:
+        with open(config_file_path, encoding="utf-8") as config_file:
```

Comment:
I would define `DEFAULT_ENCODING` and `LOCALE_ENCODING` prior to loading the configuration and use the expected pattern:

```suggestion
        with open(config_file_path, encoding=LOCALE_ENCODING) as config_file:
```